lib/Runtime/Library/JavascriptBuiltInFunctionList.h

```diff
@@ -121,7 +121,7 @@ BUILTIN(JavascriptDate, ToLocaleTimeString, EntryToLocaleTimeString, FunctionInf
 BUILTIN(JavascriptDate, ToString, EntryToString, FunctionInfo::ErrorOnNew | FunctionInfo::HasNoSideEffect)
 BUILTIN(JavascriptDate, ToTimeString, EntryToTimeString, FunctionInfo::ErrorOnNew)
 BUILTIN(JavascriptDate, ToUTCString, EntryToUTCString, FunctionInfo::ErrorOnNew)
-BUILTIN(JavascriptDate, ToGMTString, EntryToGMTString, FunctionInfo::ErrorOnNew)
+BUILTIN(JavascriptDate, ToGMTString, EntryToUTCString, FunctionInfo::ErrorOnNew) // Spec stipulates toGMTString must be the same function object as toUTCString
```

Comment:
For other cases where we have the same function object stored under multiple named properties, we don't have BUILTIN entries for both - only the one matching the entry function. I don't know if there's a requirement or not but you should make sure we need to have this entry here.

You can check out `Array.prototype.toStringLocale` or `Array.prototype.toString` for an example. Both are shared between %TypedArray%.prototype and Array.prototype but there are BUILTIN entries only for the Array methods.